src/tracks.c

```diff
@@ -2419,22 +2419,22 @@ UNUSED void func_8002AC00(s32 arg0, s32 arg1, s32 arg2) {
     }
 }
 
-// These types are probably wrong because the vars are likely still unidentified structs, but the code matches still.
-UNUSED void dayGetTrackFade(s32 *arg0, s32 *arg1, s32 *arg2) {
-    *arg0 = (unsigned) D_8011D378;
-    *arg1 = (unsigned) D_8011D370;
-    *arg2 = (unsigned) D_8011D374;
+UNUSED void get_collision_candidate_data(s32 *numCollsionCandidates, s32 **collisionCandidates,
+                                         s8 **collisionSurfaces) {
+    *numCollsionCandidates = gNumCollisionCandidates;
+    *collisionCandidates = gCollisionCandidates;
+    *collisionSurfaces = gCollisionSurfaces;
 }
 
-void func_8002ACC8(s32 arg0) {
-    D_8011B0F4 = arg0;
+void set_collision_mode(s32 mode) {
+    gCollisionMode = mode;
 }
 
-s32 func_8002ACD4(f32 *arg0, f32 *arg1, f32 *arg2) {
-    *arg0 = D_8011B0E4;
-    *arg1 = D_8011B0E8;
-    *arg2 = D_8011B0EC;
-    return D_8011B0F0;
+s32 get_collision_normal(f32 *outX, f32 *outY, f32 *outZ) {
```

Comment:
Does this still match if these three vars are merged into one Vec4f?

Reply:
Nope, it doesn't.